riotfile.py

```diff
@@ -40,6 +40,14 @@ def select_pys(min_version=min(SUPPORTED_PYTHON_VERSIONS), max_version=max(SUPPO
                 "pygments": latest,
             },
         ),
+        Venv(
+            pys=3,
+            name="mypy",
+            command="mypy {cmdargs}",
```

Comment:
should we also run as `mypy --py2` ?

Reply:
Good idea

Reply:
This does not work because
```
.riot/.venv_py392/lib/python3.9/site-packages/tenacity/_asyncio.py:36: error: invalid syntax  [syntax]
```

I'm not sure I can fix it actually. I'm leaning towards keeping this Py3 only…